Source/Core/Core/NetPlayClient.cpp

```diff
@@ -833,9 +833,9 @@ void NetPlayClient::OnStartGame(sf::Packet& packet)
 
     for (auto& device : m_net_settings.m_EXIDevice)
     {
-      int tmp;
+      ExpansionInterface::TEXIDevices tmp;
       packet >> tmp;
-      device = static_cast<ExpansionInterface::TEXIDevices>(tmp);
+      device = tmp;
```

Comment:
Wouldn't it be possible to get rid of the temporary variable here and in a few other places now?

Reply:
Ah, true, good catch